zipkin-cassandra-core/src/main/java/org/twitter/zipkin/storage/cassandra/Repository.java

```diff
@@ -707,6 +744,131 @@ private String debugInsertTraceIdByAnnotation(ByteBuffer annotationKey, long tim
                 .replace(":ttl_", String.valueOf(ttl));
     }
 
+    private class DurationRow {
+        Long trace_id;
+        Long duration;
+        Long timestamp;
+        DurationRow(Row row) {
+            trace_id = row.getLong("tid");
+            duration = row.getLong("d");
+            timestamp = row.getLong("ts");
+        }
+        public String toString() {
+            return String.format("trace_id=%d, duration=%d, timestamp=%d", trace_id, duration, timestamp);
+        }
+    }
+
+    /**
+     * @return Returns a map of trace id -> timestamp
```

Comment:
supernit, but usually I inline doc like this: `/** Returns a map of trace id -> timestamp */`